taxi/forms.py

```diff
@@ -39,6 +39,39 @@ def clean_license_number(self):
         return validate_license_number(self.cleaned_data["license_number"])
 
 
+class DriverSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Search by username"}
+        )
+    )
+
+
+class ManufacturerSearchForm(forms.Form):
+    username = forms.CharField(
+        max_length=255,
+        required=False,
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Search by name"}
+        )
+    )
```

Comment:
The field name `username` in `ManufacturerSearchForm` seems incorrect. It would be more appropriate to use `name` since the placeholder suggests searching by name.